### ISSUE TITLE
gproc 0.6 on hex.pm

### ISSUE THREAD

**Issue by benoitc** [uwiger/gproc#107]
Seems like you forgot to publish the version 0.6  to hex.pm, would it be possible to have it?

**Comment by uwiger** [comment]
Yeah, I just haven't gotten into the habit of using or publishing to hex. I'll try to do better in the future.

**Comment by jamesotron** [comment]
This issue is 22 days old and still no version 0.6 on hex. Is there any way we can make this easier for you?

**Comment by OvermindDL1** [comment]
0.6.0 would be nice, but for now everyone can just use this repo directly:
```elixir
{:gproc, git: "https://github.com/uwiger/gproc.git", tag: "0.6"}
```
Updating hex would still be very nice...  ^.^

**Comment by jamesotron** [comment]
@OvermindDL1 which is what I've done in [PhaseDB](https://github.com/jamesotron/phasedb) but for other projects which depend on PhaseDB Hex doesn't automatically include this dependency.

**Comment by OvermindDL1** [comment]
Wait it doesn't?  Is that a mix bug or a design decision to not bring in git links?

**Comment by jamesotron** [comment]
no idea.

**Comment by uwiger** [comment]
Apologies for dragging my heels on this, but I've been pretty busy lately. PRs are highly appreciated, e.g. for transitioning gproc to rebar3, making the recommended additions to the .app file etc.

**Comment by sivsushruth** [comment]
Any updates folks ? It will be handy to have newer version on Hex.

**Comment by uwiger** [comment]
Gproc 0.6.1 published to hex. Sorry about the delay.
